docs/source/accelerators/hpu.rst

```diff
@@ -112,6 +112,50 @@ This enables advanced users to provide their own BF16 and FP32 operator list ins
 
 For more details, please refer to `PyTorch Mixed Precision Training on Gaudi <https://docs.habana.ai/en/latest/PyTorch_User_Guide/PyTorch_User_Guide.html#pytorch-mixed-precision-training-on-gaudi>`__.
 
+
+Enabling Models with Convolution layers
+---------------------------------------
+Convolution operations are central to vision topologies like ResNet. Gaudi HW performs convolution operations with filter (weights) in filters last format - RSCK format where:
+R = height of the filter; S = width of the filter; C = number of channels per filter; K = number of filters
```

Comment:
Can we use the terms PyTorch uses in its docs for this: KCHW?

Reply:
This is specifically for the filters. I'll see if I can add some more document for data formats too